src/main/kotlin/simulator/lotto/Lotto.kt

```diff
@@ -1,40 +1,11 @@
 package simulator.lotto
 
-class Lotto(numbers: Set<Int>) {
-    val numbers: Set<Int>
-
-    init {
-        require(numbers.size == NUMBERS_COUNT) {
-            "로또 번호는 ${NUMBERS_COUNT}개의 숫자로 구성되어야 합니다"
-        }
-
-        require(numbers.maxOf { it } <= MAX_NUMBER) {
-            "로또 번호는 ${MAX_NUMBER}이하이어야 합니다"
-        }
-
-        require(numbers.minOf { it } >= MIN_NUMBER) {
-            "로또 번호는 ${MIN_NUMBER}이상이어야 합니다"
-        }
-
-        this.numbers = numbers.toSortedSet()
-    }
-
-    fun match(lotto: Lotto): Int {
-        var matches = 0
-        lotto.numbers.forEach {
-            if (numbers.contains(it)) matches++
-        }
-        return matches
-    }
-
+class Lotto(val number: Number) {
     override fun toString(): String {
-        return numbers.joinToString(",")
+        return number.values.joinToString(",")
     }
 
     companion object {
         const val PRICE = 1000
```

Comment:
```suggestion
        const val PRICE = 1_000
```
코틀린에서는 천단위로 언더바를 붙일 수 있어요.